libcontainer/process_linux.go

```diff
@@ -487,6 +487,15 @@ func (p *initProcess) startTime() (uint64, error) {
 }
 
 func (p *initProcess) sendConfig() error {
+	if p.config.Config.Hooks != nil {
```

Comment:
Should we just send the OCI state regardless of if there are hooks? This might be a "gotcha" moment in the future when people try to use that state and realize it's nil.

cc @cyphar

Reply:
IMO it makes more sense just to always send it.

Reply:
Addressed as a new function:
- https://github.com/opencontainers/runc/pull/2229/files#diff-e6cd3268ea1b2c0ee41532976972b36fR353-R355
- https://github.com/opencontainers/runc/pull/2229/files#diff-e6cd3268ea1b2c0ee41532976972b36fR492-R501

Marking resolved if that's ok with you :)